OJS.Workers.ExecutionStrategies/Helpers/MySqlStrategiesHelper.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace OJS.Workers.ExecutionStrategies.Helpers
+{
+    using System;
+    using System.Text;
+    using System.Text.RegularExpressions;
+
+    public static class MySqlStrategiesHelper
+    {
+        private const string InsertIntoTableRegexPattern = @"insert\s+into\s+([^(]+)\s+\([^(]+\)\s+values\s*";
+
+        public static string TryOptimizeQuery(string query)
+        {
+            if (string.IsNullOrWhiteSpace(query))
+            {
+                return query;
+            }
+
+            var newQuery = new StringBuilder();
+
+            var lines = query.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries);
+            var insertStatementRegex = new Regex(InsertIntoTableRegexPattern);
+
+            for (var i = 0; i < lines.Length; i++)
+            {
+                var currLine = lines[i];
```

Comment:
IMO, we should not use the prefix `curr` as it has a special meaning in Bulgarian...